unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -134,15 +134,15 @@ public void AddEntityOnKernel(DecentralandEntity entity, ParcelScene scene)
 
             if (keyValuePairBaseDisposable.Value.GetClassId() == (int) CLASS_ID.NFT_SHAPE)
             {
-                NFTComponent nFTComponent = new NFTComponent();
+                NFTComponent nftComponent = new NFTComponent();
                 NFTShape.Model model = (NFTShape.Model) keyValuePairBaseDisposable.Value.GetModel();
 
-                nFTComponent.color = new ColorRepresentation(model.color);
-                nFTComponent.assetId = model.assetId;
-                nFTComponent.src = model.src;
-                nFTComponent.style = model.style;
+                nftComponent.color = new ColorRepresentation(model.color);
+                nftComponent.assetId = model.assetId;
+                nftComponent.src = model.src;
+                nftComponent.style = model.style;
 
-                componentPayLoad.data = nFTComponent;
+                componentPayLoad.data = nftComponent;
```

Comment:
"payload" is a single word, `componentPayLoad` should be named `componentPayload`